### PR TITLE
MODORDERS-124 removed /orders endpoint implementation

### PR DESCRIPTION
**Pull request by grabsefx**
## Purpose
Getting collection of purchase orders already implemented in mod-orders module. So duplicated implementation should be removed from mod-orders-storage
## Approach
Removing RAML file defined for "/orders" endpoint,  removing implementation and unit tests.

### CODE REVIEW

File: descriptors/ModuleDescriptor-template.json [hunk 1]
```diff
@@ -402,11 +402,6 @@
           "methods": ["DELETE"],
           "pathPattern": "/orders-storage/purchase_orders/{id}",
           "permissionsRequired": ["orders-storage.purchase_orders.item.delete"]
-        },
-        {
-          "methods": ["GET"],
-          "pathPattern": "/orders",
```
Comment (craigmcnally) [inline descriptors/ModuleDescriptor-template.json]: should be changed to /orders-storage/orders

File: descriptors/ModuleDescriptor-template.json [hunk 2]
```diff
@@ -402,11 +402,6 @@
           "methods": ["DELETE"],
           "pathPattern": "/orders-storage/purchase_orders/{id}",
           "permissionsRequired": ["orders-storage.purchase_orders.item.delete"]
-        },
-        {
```
Comment (craigmcnally) [inline descriptors/ModuleDescriptor-template.json]: split this into its own interface `orders-storage.orders`

File: descriptors/ModuleDescriptor-template.json [hunk 3]
```diff
@@ -402,11 +402,6 @@
...
-          "methods": ["GET"],
-          "pathPattern": "/orders",
-          "permissionsRequired": ["orders-storage.purchase_orders.collection.get"]
```
Comment (craigmcnally) [inline descriptors/ModuleDescriptor-template.json]: rename to `orders-storage.orders.collection.get`

**Comment by grabsefx** [comment]
All requested changes will be done as part of https://issues.folio.org/browse/MODORDSTOR-22

**Pull request closed without merging**